applications/matter_weather_station/README.rst

```diff
@@ -175,6 +181,23 @@ The ``build_thingy53_nrf5340_cpuapp`` parameter specifies the output directory f
 
       File not found: ./ncs/nrf/applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/prj_shell.conf
 
+Generating factory data
+=======================
+
+You generate new factory data set when building for the target board by invoking the following command:
```

Comment:
This won't work without some background info, like this from 2.1.1 release, but adapted to 2.1.99:
"You can program these files for example to compare your certification test data with the factory data that passed the certification event."

Reply:
As discussed F2F I just added link to build types section.